### PR TITLE
finished note-component

### PR DESCRIPTION
**Pull request by NoahBovee**

### CODE REVIEW

**Review by gasparsaldanas — APPROVED** [review]
looks good 👍

File: src/components/CreateArea.jsx [hunk 1]
```diff
@@ -1,25 +1,62 @@
...
+  }
+
+  const submitButton = (event) => {
+    onAdd(note);
+    setNote({
+      title: "",
+      content: "",
+    });
+    event.preventDefault();
+  };
```
Comment (CCornez) [inline src/components/CreateArea.jsx:27]: should check if note is empty. Should only allow to add note when title and content are not empty

Comment (CCornez) [inline src/components/CreateArea.jsx]: should check if note is empty. Should only allow to add note when title and content are not empty

**Review by CCornez — CHANGES_REQUESTED** [review]
Looks good, add a bit of validation and then we can merge it ;)

**Comment by CCornez** [comment]
[Note component](https://trello.com/c/FxkLy5Sn/79-note-component)

**Review by CCornez — APPROVED** [review]
very nice 😎

**Pull request merged**